### DISCUSSION TITLE
Hang up Button...

### DISCUSSION THREAD

**Discussion by martykazmaier** [mkrueger/icy_term#37] [General]
It brings up the dial dialogue, but doesnt' close the connection.

**Comment by mkrueger** [comment]
It does both. At least it should do. 
You can escape out of the dialing directory dialo and try.

Which version are you using?

**Comment by martykazmaier** [comment]
.6.4

**Comment by martykazmaier** [comment]
Oh, there's no indication that it hangs up, but it does.

**Comment by mkrueger** [comment]
It's in the title - the connection time isn't displayed. It's a bit subtle - I agree.
But I drop the connection there.
